Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -593,6 +599,9 @@ open class NavigationMapView: UIView {
         lineLayer.layout?.lineJoin = .round
         lineLayer.layout?.lineCap = .round
 
+        if let gradientStops = routeLineGradient(route, fractionTraveled: 0.0, isMain: false) {
```

Comment:
How does Mapbox Navigation SDK for Android handles this case? It'd be good to provide the ability to control such feature in case if user doesn't want to show congestion segments.

Reply:
I see. I'll look into this part.

Reply:
Thanks for the suggestion! I added a public variable as `NavigationMapView.showsAlternativeCongestion`, which is default to `false`. When it set as `true` , it would allow the alternative route line show the congestion colors.